packages/grafana-ui/src/components/uPlot/config.ts

```diff
@@ -192,6 +192,24 @@ export interface StackingConfig {
   group?: string;
 }
 
+/**
+ * @alpha
+ */
+export enum GraphTresholdsStyleMode {
+  Off = 'off',
+  Line = 'line',
+  Area = 'area',
+  LineAndArea = 'line+area',
+  Series = 'series',
```

Comment:
What is the Series option?

Reply:
when the line get's threshold colors

Reply:
ah, I see. It's just not exposed in the UI, right?

Reply:
not impemented yet